src/main/java/com/example/application/ui/horizontal/ourCompany/AboutUsView.java

```diff
@@ -2,125 +2,54 @@
-        initializeUploadContainer();
-        imagesManager.setOneImage(true);
+        setData();
     }
 
     private void setData(){
-        pageEntity = pageService.findPageById(4);
+        PageEntity pageEntity = pageService.findPageById(5);
 
         pageTitle = new H1(pageEntity.getTitle());
-        pageContent = new Paragraph(pageEntity.getContent());
+        pageContent = new Paragraph();
+        pageContent.setId("pageContentWelcome");
+        pageContent.setText((pageEntity.getContent()));
         pageContent.getElement().setProperty("innerHTML", pageEntity.getContent());
 
         this.add(pageTitle, pageContent);
     }
-
-    private void setTimeline(){
-        PureTimeline timelineBSAG = new PureTimeline();
-        PureTimelineItem item1876 = new PureTimelineItem("1876", new Paragraph("Betrieb der Aktiengesellschaft Bremer Pferdebahn wurde aufgenommen"));
-        PureTimelineItem item1889 = new PureTimelineItem("1889", new Paragraph("Erste elektrische Versuchsstrecke wurde geplant"));
-        PureTimelineItem item1892 = new PureTimelineItem("1892", new Paragraph("Bremer Pferdebahn nannte sich um in Bremer Straßenbahn und begann mit dem elektrischen Betrieb"));
-        PureTimelineItem item1910 = new PureTimelineItem("1910", new Paragraph("Der erste Oberleitungsbus wurde in Betrieb genommen"));
-        PureTimelineItem item1924 = new PureTimelineItem("1924", new Paragraph("Die erste Linie des Diesel-Omnibusnetzes der Bremer Vorortbahnen GmbH entstand"));
-        PureTimelineItem item1953 = new PureTimelineItem("1953", new Paragraph("Straßenbahn übernimmt den Linienbusverkehr der BVG"));
-        PureTimelineItem item1961 = new PureTimelineItem("1961", new Paragraph("Letzte Oberleitungs-Buslinie wird eingestellt"));
-        timelineBSAG.add(item1876, item1889, item1889, item1892, item1910, item1924, item1953, item1961);
-
-        timelineBSAG.setId("timelineBSAG");
-        item1876.setId("item1876");
-        item1961.setId("item1961");
-
-        this.add(timelineBSAG);
-    }
-
-    private void initializeImagesManager(){
-        imagesManager = new ImagesManager(pageEntity.getImages(), imageService);
-        imagesManager.setImagesEntities(pageEntity.getImages());
-        imagesManager.setAllImageEntitiesData(pageEntity, userEntity);
-
-        imagesManager.initializeAllImages();
-    }
-
-    private void initializeImages(){
-        imagesContainer = new Div();
-        for(Image image : imagesManager.getImages()) imagesContainer.add(image);
-        bigContainer.add(imagesContainer);
-        bigContainer.add(imagesManager);
-    }
-
-    private void initializeBigContainer(){
-        bigContainer = new Div();
-        initializeImages();
-    }
-
-    private void initializeUploadContainer(){
-        imagesManager.initializeUploadContainer();
-        imagesUploader = imagesManager.getImagesUploader();
-        this.add(bigContainer);
-        this.add(imagesUploader);
-    }
```

Comment:
Dieser Code bleibt in der Klasse drin. Wird zwar nicht aus der DB gelesen, aber zum Zeigen bleibt er erstmal da.